### ISSUE TITLE
Neovim cannot toggle comments

### ISSUE THREAD

**Issue by TimeTravelPenguin** [Myriad-Dreamin/tinymist#511]
**Describe the bug**
I am not 100% sure if this is a bug or if I have messed up my neovim setup, so I hope this is the correct place to ask.

I am using [NvChad](https://github.com/NvChad/NvChad), and my setup doesn't follow the install guide of tinymist exactly. I am relatively new to nvim, too.

I believe I have things working, but for some reason, I cannot seem to get my `space+/` keybind to toggle comments. I get an error: `Option 'commentstring' is empty.`

In my lspconfig.lua configuration file, I have:

```lua
local configs = require "nvchad.configs.lspconfig"
local on_attach = configs.on_attach
local on_init = configs.on_init
local capabilities = configs.capabilities

local lspconfig = require "lspconfig"

lspconfig.tinymist.setup {
  on_attach = on_attach,
  capabilities = capabilities,
  single_file_support = true,
  root_dir = function()
    return vim.fn.getcwd()
  end,
  settings = {
    exportPdf = "onSave",
    outputPath = "$dir/$name",
    formatterMode = "typstyle",
  },
}
```

I do not use mason-lspconfig, so this is my best attempt to make it work with only mason and lspconfig. If anyone can confirm whether or not my setup is the root cause, I would be greatly appreciative.

Thanks

**Package/Software version:**

tinymist extension version: `v0.11.1`.
```plain
tinymist
Build Timestamp:     2024-08-04T19:30:39.635429000Z
Build Git Describe:  v0.11.18
Commit SHA:          fb0a300bc2ef8cfb74a62100441fa4a3f648f4b2
Commit Date:         None
Commit Branch:       None
Cargo Target Triple: aarch64-apple-darwin
Typst Version:       0.11.1
```

**Comment by Myriad-Dreamin** [comment]
lsp in neovim doesn't take responsibility to toggle comments, so it is probably not an issue that should report to lsp. The error "Option 'commentstring' is empty." could be a hint to update Neovim's language configuration for typst.

**Comment by TimeTravelPenguin** [comment]
Thanks for your response. I have been looking into it, but progress is slow since I don't fully understand everything about neovim.

> The error "Option 'commentstring' is empty." could be a hint to update Neovim's language configuration for typst.

As far as I can tell, I have everything up to date. Is there anything immediate that I can do to look further into this?

Thanks

**Comment by TimeTravelPenguin** [comment]
I can confirm that I do not have typst's ftplugin file for some reason. It appears to have been added to the nvim repo 3 weeks ago, so if it was released with nvim 10.1, I am unsure why I do not have it.

**Comment by TimeTravelPenguin** [comment]
Ok, it seems it was a [nightly release](https://github.com/neovim/neovim/commit/a0fd51c1e2db17ab21e2a2c907b232d05dfffc88#diff-4c2061f1e84715ff4f5877fbc49ee677ccfbfadff45409058b2504da7672ceda). Apon adding these files to my vim directory, things are working as expected. I suppose they will be released with the next stable build.

Thanks for your time.
